martin-ui/src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import martinCover from './assets/martin-cover.png'
```

Comment:
this image is somewhat incorrect - martin is not only focused on large databases... Not sure what to say there - maybe even leave it at empty-ish, or use the martin's main documentation string?